src/app/collective-rewards/rewards/hooks/useGetLastCycleDistribution.ts

```diff
@@ -0,0 +1,36 @@
+import { Cycle } from '@/app/collective-rewards/metrics'
+import {
+  RewardDistributionFinishedEventLog,
+  useGetRewardDistributionFinishedLogs,
+} from '@/app/collective-rewards/rewards'
+
+type Log = RewardDistributionFinishedEventLog[number] & { timeStamp: number }
+export const useGetLastCycleDistribution = ({ cycleStart, cycleNext }: Cycle) => {
+  const {
+    data: rewardDistributionFinished,
+    isLoading: rewardDistributionFinishedLoading,
+    error: rewardDistributionFinishedError,
+  } = useGetRewardDistributionFinishedLogs()
+
+  const isLoading = rewardDistributionFinishedLoading
+  const error = rewardDistributionFinishedError
```

Comment:
NIT: no need to define them here, as they can be defined when returning the object